index.html

```diff
@@ -130,132 +108,90 @@ <h3 class="heading">Professional Art Printing Data</h3>
-                        >Ruby</span
-                      >
+                    <div class="skill-portfolio">
+                      <span class="badge-1">HTML</span>
+                      <span class="badge-1">Bootstrap</span>
+                      <span class="badge-1">Ruby</span>
                     </div>
                     <div class="project-see">
                       <h1 class="see-project">see project</h1>
                     </div>
                   </div>
                 </div>
-              </div>
-              </div>
-                <div class="row-portfolio">
-                   <div class="card-portfolio">
-                      <div class="card-4">
+                    <div class="card-4">
                     <div class="card-profession">
                       <h3 class="heading">Professional Art Printing Data</h3>
                       <p class="card-text-portfolio">
                         A daily selection of privately personalized reads; no accounts
                         or sign-ups required. has been the industry's standard
                       </p>
                       <div class="skill-portfolio">
-                        <span class="badge"
-                          >HTML</span
-                        >
-                        <span class="badge"
-                          >Bootstrap</span
-                        >
-                        <span class="badge"
-                          >Ruby</span
-                        >
+                        <span class="badge-1">HTML</span>
+                        <span class="badge-1">Bootstrap</span>
+                        <span class="badge-1">Ruby</span>
                       </div>
                       <div class="project-see">
                         <h1 class="see-project">see project</h1>
                       </div>
                     </div>
                   </div>
-                </div>
-                </div>
-                <div class="row-portfolio">
-                  <div class="card-portfolio">
-                    <div class="card-5">
+                  <div class="card-5">
+                    <div class="card-profession">
+                      <h3 class="heading">Professional Art Printing Data</h3>
+                      <p class="card-text-portfolio">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard
+                      </p>
+                      <div class="skill-portfolio">
+                        <span class="badge-1">HTML</span>
+                        <span class="badge-1">Bootstrap</span>
+                        <span class="badge-1">Ruby</span>
+                      </div>
+                      <div class="project-see">
+                        <h1 class="see-project">see project </h1>
+                      </div>
+                    </div>
+                  </div>
```

Comment:
- [ ] As the previous code reviewer mentioned, you should keep the mobile version intact and there is still some overlapping going on in the project cards both in mobile and desktop versions. Kindly fix that.
Your design:
![image](https://user-images.githubusercontent.com/61919617/178024113-cb6f9bf7-3d22-475f-aba0-8ae35bcffbff.png)